src/main/User/CreateDeveloper.java

```diff
@@ -0,0 +1,106 @@
+package User;
+
+import Config.DeploymentLevel;
+import Config.MongoConfig;
+import Security.EncryptionObjectController;
+import Security.SecurityUtils;
+import Validation.ValidationException;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Scanner;
+
+import static com.mongodb.client.model.Filters.eq;
+
+public class CreateDeveloper {
+
+  public static void main(String[] args)
```

Comment:
Please do not make someone create a developer through the terminal line. Please refactor this

Reply:
There is no other way I can think of

Reply:
well, there should be a way to create a developer account through the new developer dashboard right?

Reply:
No, because you have to create a developer account as a developer -> how do you access the dashboard to begin with on the website -> it should not be accessible to anyone except developers.